t/perf/p2081-parallel-checkout-sparse.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+
+test_description='performance test for sparse-checkout using parallel-checkout'
+
+. ./perf-lib.sh
```

Comment:
These perf tests make me think that it would be good to have a "matrix test" that configured different settings instead of just testing different commits. But that's an aside.

Reply:
Yeah, I'm open to suggestions here.